src/components/Checkout/DonateRepaymentsToggle.vue

```diff
@@ -1,29 +1,69 @@
 <template>
-	<div v-if="showToggle" class="donate-repayments-toggle">
-		<kv-checkbox
-			class="donate-repayments-label"
-			id="donate-repayments"
-			:checkbox-right="true"
-			v-if="!myDonateRepayments"
-			v-model="donateRepayments"
-			@change="toggleDonateRepayments"
+	<div>
+		<div
+			v-if="showToggle"
+			class="tw-mt-2"
+			:class="{ 'tw-flex' : checkboxRight }"
 		>
-			<span
+			<label
+				v-if="checkboxRight"
+				for="donateRepayments"
 				id="donate-repayments-tooltip"
-				class="tw-text-small tw-text-link tw-font-medium"
+				class="tw-flex-1 tw-pt-0.5 md:tw-pt-0 tw-text-base
+					tw-font-medium tw-text-left md:tw-text-right tw-mr-2
+					tw-cursor-pointer"
+				@click="toggleCheckbox"
 			>
 				Donate loan repayments instead?
-			</span>
-		</kv-checkbox>
-		<kv-tooltip controller="donate-repayments-tooltip" theme="mint">
-			<template #title>
-				Thanks for your support!
-			</template>
-			When you check this box, repayments go back to Kiva in the form of donations,
-			helping us cover operating costs and reach even more borrowers worldwide.
-			<br>
-			Repayments from selected loans will not be added back to your account as Kiva credit.
-		</kv-tooltip>
+			</label>
+			<kv-checkbox
+				class="tw-relative tw-cursor-pointer md:tw-text-right kv-checkbox"
+				:class="{ 'tw-pt-0 ' : checkboxRight }"
+				id="donateRepayments"
```

Comment:
This id is a clever workaround to hook into your label element above! There's one catch, which is that currently `kv-checkbox` generates a [random id for itself](https://github.com/kiva/kv-ui-elements/blob/main/%40kiva/kv-components/vue/KvCheckbox.vue#L10), so this id doesn't get applied to the checkbox.

I think we can still make all this work by changing your `<label>` element above to a `<button>`.  Then I'd add a ` aria-hidden="true"` to hide that button from screenreaders so they don't hear "Donate loan repayments instead?" twice.

Reply:
Thanks @ryan-ludwig